graphqlapi/src/main/java/io/stargate/graphql/GraphqlActivator.java

```diff
@@ -52,6 +58,7 @@ public GraphqlActivator() {
   @Override
   @Nullable
   protected ServiceAndProperties createService() {
+    healthCheckRegistry.get().register("graphql", graphqlHealthCheck);
```

Comment:
will it be evaluated every time the `/health` endpoint is called?

Reply:
Yes, the `HealthCheck.check()` method is invoked every time the URI is accessed.